spec/controllers/collections_controller_spec.rb

```diff
@@ -78,6 +83,19 @@
       expect(response.body).to include(I18n.t('collections.show.no_other_user'))
     end
 
+    context 'when there are pending invites' do
+      before do
+        2.times do
+          create(:message, sender: user, recipient: create(:user), param_type: 'collection', param_id: collection.id, text: 'Invitation')
```

Comment:
With factory_bot, `create_list` would be the built-in way to achieve this ;)

Reply:
Is there a way to make sure that `create(:user)` is called each message individually?

Reply:
Sorry, I don't get the question. What's the purpose?

Reply:
I want to create two messages, and we have a uniqueness validation for the message if it is an invitation, ensuring that each recipient can only receive one invitation per collection:
```ruby
validates :param_id, uniqueness: {scope: %i[recipient_id param_type]}, if: -> { param_type == 'collection' }
```
Therefore each message would need its own user. If I replace the current `create` with a `create_list` the `create(:user)` is evaluated first and both messages have the same user, which fails the validation.